code/game/objects/items/devices/uplinks.dm

```diff
@@ -62,23 +62,22 @@ A list of items and costs is stored under the datum of every game mode, alongsid
 		index++
 		dat += "<b>[category]</b><br>"
 
-		var/i = 0
+		// First, let'
```

Comment:
Hmm

Reply:
Let A be a non-negative integer ?
Let my people be free ?

Reply:
First, let's dispel this fiction once and for all that Barack Obama doesn't know what he's doing. He knows exactly what he's doing; he's undergoing a systematic effort to change this country and make America more like the rest of the world.